src/EventStore.Core/Services/PersistentSubscription/PersistentSubscriptionService.cs

```diff
@@ -235,8 +231,8 @@ string user
 				onFail(startFromValidationError);
 				return;
 			}
-
-			CreateSubscriptionGroup(eventSource,
+			
+			var result = CreateSubscriptionGroup(eventSource,
```

Comment:
suggestion:
1. we can rename `CreateSubscriptionGroup` to `TryCreateSubscriptionGroup` 
2. [here](https://github.com/EventStore/EventStore/blob/0f4a2db987faeec93d642c2dcc6cf7d63b54837f/src/EventStore.Core/Services/PersistentSubscription/PersistentSubscriptionService.cs#L1028), we can log a warning when TryCreateSubscriptionGroup() returns false:
```
var key = BuildSubscriptionGroupKey(eventSource.ToString(), groupName);
Log.Warning("A duplicate persistent subscription: {subscriptionKey} was found in the configuration. Ignoring it.", key);
```